### ISSUE TITLE
Search content

### ISSUE THREAD

**Issue by kamov** [kolber/stacey#146]
Hi,
is possible to have a search content?
I don't see this on demo.

Thanks

**Comment by pbatts** [comment]
Yes, Stacey has a Search function. With Stacey 3.0.0, there is a Search folder in Content that contains the txt file and there is a corresponding Search template in Templates.

You can access the Search function by adding <code>/search</code> to the url. For example http://stacey/search

**Comment by kamov** [comment]
thanks for your reply.
I check but error 404: http://www.staceyapp.com/search/

**Comment by pbatts** [comment]
Try http://www.my_stacey_website.com/search. Where www.my_stacey_website.com is the url to your website. This is only for version 3.0.0.
